### DISCUSSION TITLE
How to donate without App Store subscription?

### DISCUSSION THREAD

**Discussion by dreirund** [osmandapp/OsmAnd#17593] [General]
Ahoj,

I want to support the development of OsmAnd. How can i donate without the way via an app store?

I am on‌ Android, I have no google services, no google account (and I will not make one), and I use OsmAnd via f-droid.

With that I can already use all features of OsmAnd, except the cloud backup (and sadly the support already told me that the only option to subscribe to cloud backup is via app store, which is out of question for me).

How can I‌ financially support the development of OsmAnd without having to use some proprietary third party subscription like google?

Ideally a SEPA bank transfer, second ideally credit card, third ideally PayPal (is also third party, but that I have).

If you, OsmAnd team, do only support proprietary App Stores, then I am sorry, you won't get money from me then, and it feels that you don't want money from people who are sensitive with regards to subscription to and installation of proprietary third-party services.

There is also an issue involving this: https://github.com/osmandapp/OsmAnd/issues/14672

Regards!

**Comment by jedenastka** [comment]
I also want a way to donate to this project, as OSM without a client such as OsmAnd would not be much more useful than a paper map (of course, OsmAnd without OSM would be even less useful, it works both ways).

The reasons for me are a bit different, as while I am not as privacy conscious as to avoid Google Play completely (though I certainly do not like it and only use it via Aurora Store and the web client), I do not want Google to take a cut from this purchase I will not even be using (as I'm as well using the unofficial (?) F-Droid build). They do not deserve it.

This also doesn't give donators the opportunity to give more (or less, if they can't afford it, but still want to give *something*) than just the base cost, or give more than one time (perhaps monthly/yearly).

**Comment by centic9** [comment]
Same for me. Would love to give back to OsmAnd without the "store" fuss. A simple "send your money to this IBAN" would be very useful for people who simply want to say "thanks".

**Comment by mattmccutchen** [comment]
As [already noted](https://github.com/osmandapp/OsmAnd/issues/17681#issuecomment-3012141854) on several other issue threads, OsmAnd now allows purchases through FastSpring (https://osmand.net/pricing). Are people happier with that option than the Google Play Store, etc.?  It looks like FastSpring is basically just acting on OsmAnd's behalf for a bunch of legal purposes and doesn't present the broader privacy exposure that Google and similar stores do.  It accepts credit cards and PayPal.

It sounds like OsmAnd is unlikely to ever accept direct bank transfers [due to regulatory hassles](https://github.com/osmandapp/OsmAnd/issues/7307#issuecomment-1012562668), so FastSpring might be about as good of a payment method as we can hope for.  It's still a pain to give custom amounts though.

**Reply by gm-333** [reply]
FastSpring wants me Create an Account and so on again. Not using it.
I donated to OSM instead - they have a paypal link.

**Reply by scaidermern** [reply]
> I donated to OSM instead - they have a paypal link.

PayPal :see_no_evil:

**Reply by gm-333** [reply]
Care to elaborate/suggest something better? Thank you.

**Reply by scaidermern** [reply]
Sorry, my comment wasn't quite constructive. However, replacing one bad US service (Google Play) with another bad one (PayPal) doesn't seem to be like a good option.

The OpenStreetMap Foundation seems to accept European SEPA transfers for donations. Yet, keep in mind that OSM and OsmAnd are different entities. Donations to the OSM Foundation are definitely a good thing but won't arrive at OsmAnd. Nevertheless OsmAnd will profit indirectly from them, to a certain extent.

**Reply by gm-333** [reply]
true, but for me paypal means one possibly trackable transaction with one login  vs. having a google account and a phone linked to that account (=playstore OSMAND), which probably means some form of constant tracking. Seems to be a no brainer decision, if understand things correctly.
And yes, they are different entities, which I understand. But as stated above, I would have to create another f* profile at another f* company just to be able to donate. I gladly go out of my way to donate anyway, but having to make an account is too much, it should be easier (ideally SEPA, CC or paypal (not because its good, but many people have it))
(... and I didn't see the SEPA option, otherwise I would have used that ;)
